core/domain/question_domain.py

```diff
@@ -71,6 +71,16 @@
 CMD_CREATE_NEW: Final = 'create_new'
 
 
+UnalteredUnionOfCustomizationArgsDictValues = Union[
```

Comment:
I don't think I understand this. What's "Unaltered" mean?

Also we should have just one "Union" type for this that's used everywhere.

Reply:
Fixed, Now only single Union type is used everywhere.